Step/Event/Action/GoNextStepEventAction.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/**
+ * @author:  Gabriel BONDAZ <gabriel.bondaz@idci-consulting.fr>
+ * @license: MIT
+ */
+
+namespace IDCI\Bundle\StepBundle\Step\Event\Action;
+
+use IDCI\Bundle\StepBundle\Form\Type\GoNextFormType;
+use IDCI\Bundle\StepBundle\Step\Event\StepEventInterface;
+use Symfony\Component\OptionsResolver\Options;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class GoNextStepEventAction extends AbstractStepEventAction
+{
+    /**
+     * {@inheritdoc}
+     */
+    protected function setDefaultParameters(OptionsResolver $resolver)
+    {
+        $resolver
+            ->setDefaults(array(
+                'condition' => true,
+                'path_index' => 1,
+            ))
+            ->setAllowedTypes('condition', array('bool', 'string'))
+            ->setAllowedTypes('path_index', array('int', 'string'))
+            ->setNormalizer('condition', function (Options $options, $value) {
+                return (bool) $value;
+            })
+            ->setNormalizer('path_index', function (Options $options, $value) {
+                return (int) $value;
+            })
+        ;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function doExecute(
```

Comment:
single line ?

Reply:
All the other step event actions were done like this, so i should keep this like that ?